src/api/ledger/accountinfo.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+const utils = require('./utils');
+const removeUndefined = require('./parse/utils').removeUndefined;
+const validate = utils.common.validate;
+const composeAsync = utils.common.composeAsync;
+
+function formatAccountInfo(response) {
+  const data = response.account_data;
+  return removeUndefined({
+    sequence: data.Sequence,
+    xrpBalance: utils.common.dropsToXrp(data.Balance),
```

Comment:
Well I think it's nice to maintain the symmetry of having a "get" method that corresponds to every "prepare" method. If we broke that symmetry, people would look for the symmetry and get confused when there are a bunch of fields in one but not the other. We could merge the settings into the "getAccountInfo" response, but then people will argue that "getSettings" is totally redundant and want to get rid of it. So I don't know, it seems like this is the simplest solution.